### PR TITLE
Browser Location (version 2)

### PR DESCRIPTION
**Pull request by capndesign**
This fixes the looping issues and is properly merged with master.

### CODE REVIEW

File: static/js/main.js [hunk 1]
```diff
@@ -87,25 +106,14 @@ app.model({
...
-          return { geolocation: geo, cachedCity: city, geoCacheTime: time }
-        } else {
-          Raven.captureMessage("Location with no city: "+response.loc, { level: 'warning' });
-        }
-      } catch(e) {
-        Raven.setExtraContext({ json: data })
-        Raven.captureMessage("Couldn't parse ipinfo json", { level: 'error' });
-      }
+    receiveIPInfoLoc: (state, data) => {
+      geo = response.loc
```
Comment (nickoneill) [inline static/js/main.js]: Where is `response` coming from? It works but I do not understand why!

**Comment by nickoneill** [comment]
Just that one comment that I want to understand, looks awesome and handles lots of different weird cases!

**Comment by capndesign** [comment]
@nickoneill I think this was a magical fat arrow that saved us! I changed `response` to `data`, so we should be good.

**Comment by nickoneill** [comment]
Awesome change! This looks great on production

**Pull request merged**